python/kserve/test/test_server.py

```diff
@@ -507,7 +507,8 @@ def test_predict_ce_avro_binary(self, http_server_client):
         writer.write(msg, encoder)
         data = bytes_writer.getvalue()
 
-        event = dummy_cloud_event(data, set_contenttype=True)
+        event = dummy_cloud_event(data)
+        event._attributes["content-type"] = "application/avro"
```

Comment:
This should be set in the dummy_cloud_event and pass in the content_type as argument

Reply:
done